src/cli.js

```diff
@@ -43,6 +43,8 @@ function buildReporter(origin) {
     };
 }
 
+exports.buildReporter = buildReporter;
```

Comment:
@hnrch02 Sorry, I'm sure I'm missing something simple here, and I don't want to be a nuisance but I'm confused about how rewire exposes the program. I don't mind using rewire but how do I expose the "program" to the tests if I don't export anything? I'll concede I'm exposing the wrong thing here and I need to expose the primary "CLI tool" but not sure how to do that here. Rewire as it says works like require, but with the ability to mock stuff. So do I need to wrap the CLI in `module.exports` like node-jscs like they do in `cli.js` or something else.

Sorry again.